versioning/scripts/cloudbuild/main.go

```diff
@@ -251,6 +296,8 @@ func main() {
 	newTagsPtr := flag.Bool("new_tags", false, "Require that image tags do not already exist.")
 	firstTagOnly := flag.Bool("first_tag", false, "Build only the first per version.")
 	timeoutPtr := flag.Int("timeout", 0, "Timeout in seconds. If not set, the default Cloud Build timeout is used.")
+	enableParallel := flag.Bool("enable_parallel", false, "Enable parallel build and bigger VM")
```

Comment:
shouldn't this be `true` by default? we still respect what's in the config file right, which by default is off

Reply:
I do want to keep this change as optional, with backward compatibility. It requires tests to parallelable, so this change can possibly break builds that depend on sequential execution.

Reply:
I understand this as: even when this flag is `true`, unless we change the config files, no parallelization will automatically happen. `forceParallel` on the other hand will break non-ready tests.

Reply:
What kind of config file you have in mind? I do not use any. Please tell me more about this feature.

In this patch logic is as follows:
- no flag - as is, serial build,
- enable flag - if 'big repo', then parallel build, else serial build,
- force flag or both flags - parallel build.

Changing enable_parallel to default true:
- no flag - if 'big repo', then parallel build, else serial build,
- negate enable flag - as is, serial build,
- force flag - parallel build.

Reply:
My bad--for whatever reason I thought our `versions.yaml` has a bit indicating whether it desires to be parallelized. I see that 'big repo' is defined in `shouldParallelize`.